NuRadioReco/modules/channelGalacticNoiseAdder.py

```diff
@@ -50,32 +73,107 @@ def begin(
         ----------
         debug: bool, default: False
             It True, debug plots will be shown
+        skymodel: {'lfmap', 'lfss', 'gsm2016', 'haslam', 'ssm', 'gmoss', 'ulsa_fdi', 'ulsa_dpi', 'ulsa_ci'}, optional
+            Choose the sky model to use. If none is provided, the Global Sky Model (2008) is used as a default.
         n_side: int, default: 4
             The n_side parameter of the healpix map. Has to be power of 2
             The radio skymap is downsized to the resolution specified by the n_side
             parameter and for every pixel above the horizon the radio noise coming
             from that direction is calculated. The number of pixels used is
             12 * n_side ** 2, so a larger value for n_side will result better accuracy
             but also greatly increase computing time.
-        interpolation_frequencies: array of float
+        freq_range: array of len=2, default: [10,1100] * units.MHZ
             The sky brightness temperature will be evaluated for the frequencies
-            in this list. Brightness temperature for frequencies in between are
+            within this limit. Brightness temperature for frequencies in between are
             calculated by interpolation the log10 of the temperature
             The interpolation_frequencies have to cover the entire passband
             specified in the run method.
+        interpolation_frequencies: array of frequencies to interpolate to.
+            Kept for historic purposes with intention to deprecate in the future.
         """
         self.__debug = debug
         self.__n_side = n_side
-        self.__interpolaiton_frequencies = interpolation_frequencies
+
+        if interpolation_frequencies is None:
+            if freq_range is None:
+                freq_range = np.array([10,1100])
+
+            # define interpolation frequencies. Set in logarithmic range from freq_range[0] to freq_range[1],
+            # rounded to 0 decimal places to avoid import errors from LFmap abd tabulated models.
+            self.__interpolation_frequencies = np.around(
+                np.logspace(
+                    *np.log10(freq_range), num=15
+                ), 0
+            ) * units.MHz
+        else:
+            self.__interpolation_frequencies = interpolation_frequencies * units.MHz
+            logger.warning("DeprecationWarning: Optional argument 'interpolation_frequencies' was replaced by 'freq_range'.")
```

Comment:
Why would one deprecate it? However you can drop the unit in l107 and add it in l99.

Reply:
> However you can drop the unit in l107 and add it in l99.

actually this is important for the same reason as the comment above.